tests/fd/FileDescriptor.test.ts

```diff
@@ -22,7 +21,7 @@ describe('File Descriptor', () => {
   const origBuffer = Buffer.from('Test Buffer for File Descriptor');
   beforeEach(async () => {
     dataDir = await fs.promises.mkdtemp(
-      path.join(os.tmpdir(), 'encryptedfs-test-'),
```

Comment:
Actually these are meant to still use `path.join` and not `pathJoin`. This is because, this is a real directory, and not inside EFS.

Reply:
Shoot, my bad. I'll fix that real quick.

Reply:
Should be done on #68.